docs/api.md

```diff
@@ -0,0 +1,40 @@
+Short API doc.
+
+# Authentication
```

Comment:
Any improvement makes sence :), but now this API is for MVP version only. We could extend it on demand, no need to design it carefully right now.